### ISSUE TITLE
A11y_AzureMachineLearningExtension_Workspaces_ScreenReader: NVDA/Narrator is not announcing the 'default' state of a workspace when focus lands on default workspace.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-tools-for-ai#1263]
**Github Tags:**
#A11yMAS;#A11yTCS;#WCAG1.3.1;#ScreenReader;#Narrator;#NVDA;#Win32;#DesktopApp;#A11ySev2;#SH-AzureMachineLearningExtension-Win32-Aug2021;#Development and Test Resources for VS and VSCode AI Tools;

### Environment Details:
Visual Studio Code

### Repro Steps:

1. Turn on Narrator/NVDA
2. Hit the Azure Machine Learning Studio URL and sign in with appropriate credentials.
3. Tab till Notebooks in the left pane and press enter. Select any file from the pane, tab till "Edit in VS code "option in command bar and press enter.
4. A pop up will appears on the screen to open VS Code app, tab till 'Open' button and press enter
5. Tab till Azure in the left pane and press enter, tab till any subscription ex: ML.Net Model Builder Demo and press enter.
6. Tab till a default workspace ex: Basic Workspace under the subscription(already pinned workspace)
7. Verify whether screen reader reads default state or not when focus lands on workspace.

### Actual Result:
NVDA/Narrator is not announcing the 'default' state of a workspace when focus lands on default workspace.

### Expected Result:
NVDA/Narrator should announce the 'default' state of a workspace when focus lands on default workspace.
Ex: Basic Workspace default The worskpace is top level etc

### User Impact:
Screen reader users will not know that a workspace is set as default if screen reader does not announce the state properly

**Comment by tbombach** [comment]
We need to set the AccessibilityInfo for the TreeNodes to improve the narrator experience. The workspace nodes accessibility info needs to be improved because it doesn't include the name of the workspace or whether it is default or not. Instead, it include a generic message saying "The workspace is top level etc"

**Comment by NiharikaTipirishetty** [comment]
issue fixed, closing the bug
#Closed; 
Narrator is announcing default state
![1263_narrator_no issue](https://user-images.githubusercontent.com/83570638/226287117-e5c54c99-10cb-4841-a266-3afbbb636d1f.png)
NVDA is announcing default state
![1263_nvda_no issue](https://user-images.githubusercontent.com/83570638/226287172-e44c85b9-ae0e-4621-811a-d565bd435962.png)
